### ISSUE TITLE
What version of TraX do CppMT use?

### ISSUE THREAD

**Issue by mirsking** [gnebehay/CppMT#35]
Hi, I'm trying to use trax to evaluate the performance of tracking algorithm.
And I find in CppMT's 'CMakeLists.txt' file, there are something about TraX.
But I don't know which version the code uses.

I tried this version, but failed.
https://github.com/votchallenge/trax

**Comment by gnebehay** [comment]
Yes, unfortunately CppMT still uses the VOT2014 version of trax. Compared to VOT2015, the trax communication protocol remained the same, but the trax interface got simplified. This means there are two options for running CppMT on VOT2015.

a) You compile the trax version of CppMT for VOT2014 using the vot2014-final release from here: https://github.com/votchallenge/vot-toolkit/releases . The compiled version then should still work without problems on VOT2015.

b) Someone updates the trax version of CppMT for VOT2015, which should not be very hard to do. 

Unfortunately I am currently very busy with finishing my PhD and looking for a new job so I cannot guarantee that I will find the time soon for doing b)

Hope this helps,
Georg

**Comment by mirsking** [comment]
Thank you very much. Then I will try a) firstly.
